ebu_tt_live/xsd/ebutt_livemessage.xsd

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns="http://www.w3.org/1999/xhtml"
```

Comment:
xhtml? Is that correct? Seems surprising not to use the XMLSchema namespace as the default and remove all the `xs:`s everywhere.